### ISSUE TITLE
CONSOLIDATE ALL PREVIOUS TAF ISSUES

### ISSUE THREAD

**Issue by colinpmillar** [ices-eg/wg_WGTAFGOV#86]
In order to start clean, it was agreed at WGTAFGOV to organise all disperate TAF related issues into one place, and then organise into milestones.

The group can then prioritise milestones rather than look through detailed issues.

**Comment by colinpmillar** [comment]
First thoughts on how to proceed are:
* first filter out the worthwhile ones, 
* group them into themes / coherent epics: UI, technical user interface (R command line etc.), database, RDBES, training
* collapse similar items where possible, 
* add labels for theme / epic / other
* WGTAFGOV to priortitise theme - or further split into milestones.

**Comment by davidcurrie2001** [comment]
I think WGTAFGOV should be coming at the problem from the other direction.  We first discuss and agree the overall priorities for 2024, then break those down into specific objectives / indicators / milestones.  This then gives you a way to evaluate the detailed issues and decided whether they should be done e.g. does the issue fit with one of the agreed priorities for 2024? Does the issue contribute towards hitting an agreed milestone / fulfilling an agreed objective?

**Comment by iagomosqueira** [comment]
I agree with David here. Do we want to start here to discuss the priorities for 2024?

**Comment by colinpmillar** [comment]
Potential priorities:
* low uptake of taf
* user interface
* database of inputs / output
* definition of input / outputs
* template workshop
* input / output taf type definition WK / subgroup
*
